tests/cancun/eip1153_tstore/test_tstorage_create_contexts.py

```diff
@@ -122,16 +122,9 @@ class InitcodeTestCases(PytestParameterEnum):
         "expected_storage": {0: 0x0000, 1: 0x0001},
     }
 
-    def __init__(self, value):
```

Comment:
Very nice that the need for this boiler plate code is eliminated! Yet, it's still possible to modify/define new values in an optional `__init__()`!